test/index.html

```diff
@@ -13,13 +13,20 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
   <title>lazy-imports tests</title>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
   <script src="../../web-component-tester/browser.js"></script>
+  <link rel="import" href="../../polymer/polymer.html">
 </head>
 <body>
   <script>
+    var imports = Polymer.Element ? [
```

Comment:
@justinfagnani @valdrinkoshi This work? Firefox wasn't playing well, any thoughts?